components/translate/core/common/brave_translate_features.h

```diff
@@ -34,6 +36,10 @@ bool ShouldUpdateLanguagesList();
 // google translate endpoint. False by default, use it only for local testing.
 bool UseGoogleTranslateEndpoint();
 
+// True if automatic translation logic is enabled.
+// This include core logic and UI element.
```

Comment:
"Includes core logic and the UI element."

Reply:
```suggestion
// Includes core logic and UI elements.
```